### ISSUE TITLE
Not getting expected results from fine-tuning of 'en_PP-OCRv5_mobile_rec' (text recognition)

### ISSUE THREAD

**Issue by jeet-Abhi123** [PaddlePaddle/PaddleX#4591]
First evaluated on my test data and it is having excellent performance. 
But I still want to improve the performance by few margin. 

- I trained it for first 120 epochs and then I evaluated, it was having poor performance than base model. 
- Then I continue training from best_accuracy model weights to 200 epochs and then I evaluated, the performance, it was again poor. 
- The validation accuracy is not increasing beyond 86-87%. I at least need to increase above 90%. 
- Command :  ```python main.py -c paddlex/configs/modules/text_recognition/en_PP-OCRv5_mobile_rec.yaml -o Global.mode=train -o Global.dataset_dir=/path/ocr_train_data -o Train.resume_path=path/PaddleX/output_2/best_accuracy/best_accuracy -o Train.epochs_iters=250```

**Comment by jeet-Abhi123** [comment]
Can we change the characters, symbols in dict.txt according to our dataset, or it has to be fixed?
Because in 'en_PP-OCRv5_mobile_rec' there are 438 chars in the ```dict.txt```.

**Comment by leo-q8** [comment]
First, you may want to test the model at this level of accuracy to see if it meets your requirements—an accuracy above 85% is already quite good. Also, could you share roughly how much data you have in your training set? The "en_PP-OCRv5_mobile_rec" model was trained on around a million samples covering most scenarios. Fine-tuning it with your own data for a specific scenario may actually lead to a drop in performance.

**Comment by jeet-Abhi123** [comment]
The base model is good, but it is less than our ocr models benchmark. It has validation accuracy of 86%, we are expecting more.

I am fine-tuning on around 2466 images of documents. The en_PP-OCRv4_mobile_rec and en_PP-OCRv5_mobile_rec has major difference in their dict.txt file. In v4 model, it has total around 96 characters but in v5 it has around 438 characters. Our fine-tuned v4 model is quite good and performing really good in our benchmarks but while fine-tuning v5 on the same dataset has seen decrease in performance on the base model as well.  Can you suggest something?

**Comment by leo-q8** [comment]
The model architecture is the same for v4 and v5, but v5 includes more characters in its dictionary due to broader coverage. We recommend customizing the dictionary for your specific needs and training with either the v4 or v5 configuration.

**Comment by jeet-Abhi123** [comment]
Ok, 
So according to your statement, we can adapt the dictionary to our use case?

**Comment by leo-q8** [comment]
Certainly. If there are characters in the official dict file that are not used in your scenario, keeping them would be a waste of model parameters. As for v4 or v5 models, as mentioned earlier, their architectures are the same, so you can choose either one. However, due to differences in pre-trained weights, their fine-tuned performance may vary in your specific use case. You can simply select the model configuration that delivers the highest accuracy for your needs.

**Comment by TingquanGao** [comment]
The issue has no response for a long time and will be closed. You can reopen or new another issue if are still confused.

---
_From Bot_
